app/controllers/outlets_controller.rb

```diff
@@ -71,10 +75,21 @@ def update
     end
     
     if @outlet.save
+      somebody_was_notified = ''
```

Comment:
using @somebody_was_notified (as an instance variable) will make this available in such a way that you wouldn't need to initialize it here,
but could still reference it below